lib/models/os.coffee

```diff
@@ -322,7 +322,8 @@ getOsModel = (deps, opts) ->
 	# @memberof resin.models.os
 	#
 	# @param {String|Number} nameOrId - application name (string) or id (number).
-	# @param {Object} [options={}] - OS configuration options to use.
+	# @param {Object} options - OS configuration options to use.
+	# @param {String} options.version - Required: the OS version of the image.
```

Comment:
* We should also update the examples.
* `resin.models.os.getConfig('MyApp', { version })` ~~seems a bit weird. Could go with it, but since we are breaking things, how about changing to use a single object param like we did in a few other places in the MC branch?~~ Didn't have in mind that it can accept other options as well... Since we are already on this, how about adding those as optional `options.param`s?

Reply:
> We should also update the examples.

Very true, fixed

> Since we are already on this, how about adding those as optional options.params?

Yeah, I'm not totally sure about this either way. Right now it doesn't quite match our API in most other places, where options are purely optional, which isn't great. I think I'd rather keep it in the options object though, for a few reasons:

* It's easier to migrate for existing users (many of whom will hopefully already be passing a version anyway)
* I expect that most people will be passing at least some other options in here anyway, not just version alone.
* This way it matches the actual API that way (which includes the version in the options object like any other option)
* With a version + a parameters option, we'd have to decide what to do if you pass a version argument normally _and_ in the parameters (since we'd be squishing them straight back into one object immediately anyway)
* Extra parameters are noisy, and I want less of them :smiley:.